packages/desktop-client/src/components/reports/disabledList.ts

```diff
@@ -94,54 +103,54 @@ const modeOptions = [
   },
 ];
 
+export function disabledGraphList(
+  item: string,
+  newGraph: string,
+  type: 'disabledSplit' | 'disabledType',
+) {
+  const [graphList] = modeOptions.filter(d => d.description === item);
+  const [disabledList] = graphList
+    ? graphList.graphs.filter(e => e.description === newGraph) || []
+    : [];
+  return disabledList[type];
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏use `find` if you need only a single item from the list.
```suggestion
  const graphList = modeOptions.find(d => d.description === item);
  
  if (!graphList) {
    return something;
  }
  
  const disabledList = graphList.graphs.find(e => e.description === newGraph);
  
  if (! disabledList) {
    return something;
  }
  
  return disabledList[type];
```